src/editors/containers/TextEditor/components/ImageSettingsModal/index.test.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { hooks } from './index';
+
+jest.mock('react', () => {
```

Comment:
oh dear... you should probably delete this file.  I am heavily re-writing this component in my PR, and have included tests for the updated hook behavior.